lib/api/v3/users/user_representer.rb

```diff
@@ -58,6 +58,22 @@ def initialize(model, options = {}, *expand)
           }
         end
 
+        link :lock do
+          {
+            href: api_v3_paths.user_lock(represented.id),
+            title: "Set lock on #{represented.login}",
+            method: :post
+          } if current_user_is_admin && represented.status == User::STATUSES[:active]
+        end
+
+        link :unlock do
+          {
+            href: api_v3_paths.user_lock(represented.id),
+            title: "Remove lock on #{represented.login}",
+            method: :delete
+          } if current_user_is_admin && represented.status == User::STATUSES[:locked]
```

Comment:
According to the specs, ``represented``can also be a Group instance, which is why the--more readable --members ``active?``, ``locked?`` are not available.

Also, as the ``/lock`` resource is accessible from admins only, their links should also be hidden from regular users.